bin/bundix

```diff
@@ -1,92 +1,97 @@
-#!/usr/bin/env ruby
+#! /usr/bin/env ruby
```

Comment:
The only place where I see this space is in the nix world. Do you know the motivation behind ?